src/App.js

```diff
@@ -0,0 +1,21 @@
+import './App.css';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import Countries from './Components/countries';
+import Nav from './Components/Navigation';
+import Details from './Components/details';
+
+function App() {
+  return (
+    <>
+      <Nav />
+      <BrowserRouter>
+        <Routes>
+          <Route path="/" element={<Countries />} />
+          <Route path="/countryDetails/" element={<Details />} />
+        </Routes>
+      </BrowserRouter>
+    </>
+  );
+}
+
+export default App;
```

Comment:
- [ ] Nice job 👏 , however, your app does not render an interctive page. its static with no links or navigation.
- Kindly review your compoent, making sure all paths and links are properly typed or inserted 👍 

### Your app on render:
![image](https://github.com/unorjikingsley/Metrics-WebApp/assets/50990925/54693a2f-9c8b-4d44-a6e8-f6983389c3c6)

Reply:
Hi @Shedrack-Sunday please i need your help to solve this issue. The API data is not being rendered in my deployed React application, but it works fine when running npm start locally?